### ISSUE TITLE
Page-Templates folders & Files

### ISSUE THREAD

**Issue by rcass** [temefleet/Inhabitent#5]
Currently the page templates has multiple files in it. Organization wise, files that should be in the page-template folders are Custom Page Templates for Global Use (aka does it create a drop down menu and user has to choose it), current the only one that is going that is the about.php.

When the page-template for one specific page, typically those are just located in the root of the theme folder. 

Basically you just need to move archive-product.php and taxonomy-product-type.php to the root of the theme (aka one directory up).

Here is more [documentation](https://developer.wordpress.org/themes/template-files-section/page-template-files/#creating-custom-page-templates-for-global-use) about it if you need!

- [x] Move  archive-product.php and taxonomy-product-type.php to theme root

**Comment by temefleet** [comment]
Done. Thanks.

**Comment by rcass** [comment]
Nice! 💯  (FYI you can totally check the checkboxes in the issues ticket!)
